### ISSUE TITLE
iperf3 not available for current release

### ISSUE THREAD

**Issue by Orv** [aredn/aredn_ar71xx#205]
On two separate nodes, both running 3.18.9.0, after refreshing the package list, and selecting iper3, got this when trying to install the package:

Unknown package 'iperf3'.
Collected errors:
 * opkg_install_cmd: Cannot install package iperf3.
Failed to restart all services, please reboot this node.

**Comment by ae6xe** [comment]
I'm not able to reproduce.  I'll try some other nodes.  I get this when  doing a "refresh", selecting "iperf3" and isntalling (on an hAP AC Lite) below.  iperf3 was functional after install. 

Package Management
--
Installing iperf3 (3.5-1AREDN) to root... Downloading http://downloads.arednmesh.org/releases/3/18/3.18.9.0/packages/mips_24kc/base/ip erf3_3.5-1AREDN_mips_24kc.ipk Configuring iperf3. Failed to restart all services, please reboot this node. | Installing iperf3 (3.5-1AREDN) to root... Downloading http://downloads.arednmesh.org/releases/3/18/3.18.9.0/packages/mips_24kc/base/ip erf3_3.5-1AREDN_mips_24kc.ipk Configuring iperf3. Failed to restart all services, please reboot this node.
Installing iperf3 (3.5-1AREDN) to root... Downloading http://downloads.arednmesh.org/releases/3/18/3.18.9.0/packages/mips_24kc/base/ip erf3_3.5-1AREDN_mips_24kc.ipk 

Configuring iperf3. 
Failed to restart all services, please reboot this node.

**Comment by Orv** [comment]
On a third node, I tried it and it worked.  I went back to the previous node and tried it again.  It found the package (image 1) but couldn't install it (image 2).  Will keep investigating,
![iperf3-1](https://user-images.githubusercontent.com/1124405/46251551-37390180-c40a-11e8-90c4-44057853faf7.png)
![iperf3-2](https://user-images.githubusercontent.com/1124405/46251552-37d19800-c40a-11e8-81de-a48474dd543e.png)

**Comment by Orv** [comment]
Well, it's like this - I had forgotten I had run those firewall Internet blocking commands you gave me, hence it did not have Internet access.  Once I rebooted that node I was able to download and install iperf3.

HOWEVER, on the adjacent DtD connected node (identical hardware and configuration) when I refreshed the package list, which was successful, iperf3 is missing.    I'll keep playing with it.

![iperf3-3](https://user-images.githubusercontent.com/1124405/46251681-dd870600-c40e-11e8-8b31-940e10404460.png)
